packages/jmix-react-antd/src/ui/form/controls/RichTextArea.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from "react";
+import { JmixFormFieldWrapper } from "./base/JmixFormFieldWrapper";
+import { JmixFormFieldProps } from "./base/JmixFormFieldProps";
+import { RichTextField } from "./RichTextField";
+import { EditorProps } from "react-draft-wysiwyg";
+import { ContentState, EditorState, RawDraftContentState } from "draft-js";
+import htmlToDraft from "html-to-draftjs";
+import draftToHtml from "draftjs-to-html";
+
+export const RichTextArea = ({
+  entityName,
+  propertyName,
+  formItemProps,
+  ...rest
+}: JmixFormFieldProps & EditorProps) => {
+  function stringToEditorState(input: string): EditorState {
+    const { contentBlocks, entityMap } = htmlToDraft(input);
+    return EditorState.createWithContent(
+      ContentState.createFromBlockArray(contentBlocks, entityMap)
+    );
+  }
+
+  function rawStateToString(state: RawDraftContentState): string {
+    return draftToHtml(state);
+  }
```

Comment:
Please move these functions outside of the component. Just put them in the same file with `RichTextField`, but outside the component. If you put them inside the component, they will be re-created on each render.